Lib/python/builtin.swg

```diff
@@ -117,7 +117,11 @@ SwigPyBuiltin_FunpackSetterClosure (PyObject *obj, PyObject *val, void *closure)
 
 SWIGINTERN void
 SwigPyStaticVar_dealloc(PyDescrObject *descr) {
+#if PY_VERSION_HEX >= 0x03060000
+  PyObject_GC_UnTrack(descr);
+#else
   _PyObject_GC_UNTRACK(descr);
```

Comment:
```suggestion
  PyObject_GC_UnTrack(descr);
```
No need for the PY_VERSION_HEX check as we support 2.7, 3.2+ and PyObect_GC_UnTrack is available in 2.7 onwards. Please resubmit the pull request with this change to make sure the failing CI tests are fixed. Thanks.